### ISSUE TITLE
Episode Listing not working

### ISSUE THREAD

**Issue by guardiaopt** [tboothman/imdbphp#318]
For the past few days, the episode listing has ceased to function correctly, as observed in this link: https://www.imdb.com/title/tt13833978/. Is anyone else experiencing the same issue?

**Comment by Thomasdouscha** [comment]
I have tried to get it and there is no issue.

guardiaopt ***@***.***>, 4 Ağu 2023 Cum, 14:58 tarihinde şunu
yazdı:

**Comment by guardiaopt** [comment]
My IP is German, but I set the language to English in the settings.

$config->language = 'en-US,en';

**Comment by duck7000** [comment]
Imdb converted this page to a new layout so episodes() no longer work

**Comment by mam4dali** [comment]
The issue has been occurring for over a week. This is a new page:
<img width="646" alt="image" src="https://github.com/tboothman/imdbphp/assets/13052531/cec816a3-75e5-413e-8e69-15a5c0608390">

HTML source of the new page:
[new source.txt](https://github.com/tboothman/imdbphp/files/12267886/new.source.txt)

**Comment by mam4dali** [comment]
I found this address to retrieve information, but I'm unsure about its reliability. There's a token inside it, and it might expire after a while:
`https://www.imdb.com/_next/data/5s-5cAOpj5F5KR73DcheJ/title/tt2560140/episodes.json?season=1&tconst=tt2560140`

**Comment by duck7000** [comment]
No need for trying to fix this, the only way to fix it is to use GraphQL
See #314 
If anybody is interested to fix imdphp you may use my code to do so

**Comment by mam4dali** [comment]
I have made some changes and will test them in the coming days. If everything goes smoothly, I will submit a merge request:
https://github.com/mam4dali/imdbphp/commit/74f2036794a183cbeb2b02cba942da37c9ad7c53

Of course, in my opinion, GraphQL is the best approach. I don't have much knowledge about it, but if someone can solve it using that method, we welcome it.

**Comment by duck7000** [comment]
@mam4dali that is exactly what i did..
did you read #314 and checked my repo?

**Comment by mam4dali** [comment]
> @mam4dali that is exactly what i did.. did you read #314 and checked my repo?

Yes, but I'm not sure if it will merge smoothly with the current repository.

**Comment by duck7000** [comment]
Possibly not, you might make some adjustments although it is almost the same
But it is a very good starting point and it works great
